Minor changes and fixes on Attribution (#2514)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/attribution-reporting/index.md

{% Aside %}

> The API does not rely on third-party cookies. However, in the testing phase, third-party cookies need to be enabled for the API to be enabled. This will make it easier to get debug reports and to compare the API's results with cookie-based attribution.

Reviewer: ```suggestion
The API does not rely on third-party cookies. However, in the testing phase, third-party cookies need to be enabled for the API to be enabled. This allows developers to get debug reports and compare the API's results with cookie-based attribution.
```

Reviewer: Just a suggestion, but not required!
